src/metaswitch/crest/test/api/homestead/provisioning/handlers/private.py

```diff
@@ -100,7 +100,7 @@ def test_put_plaintext_password(self, put_digest):
     def test_get_missing_subscriber(self, get_digest, send_error):
         get_digest.side_effect = NotFoundException()
         self.handler.get("private_id")
-        get_digest.assert_called
+        self.assertTrue(get_digest.called > 0)
```

Comment:
These seem like unrelated changes... they look fine but did you mean to include them?